### PR TITLE
pyhtonic quibbles

### PR DESCRIPTION
**Pull request by paulgmiller**
---
This is not important at all but this chunk of code felt strange to me. strip won't ever throw a value error.

This checklist is used to make sure that common guidelines for a pull request are followed.

### General Guidelines

- [ ] Have you run `azdev style <YOUR_EXT>` locally? (`pip install azdev` required)
- [ ] Have you run `python scripts/ci/test_index.py -q` locally?

For new extensions:

- [ ] My extension description/summary conforms to the [Extension Summary Guidelines](https://github.com/Azure/azure-cli/blob/dev/doc/extensions/extension_summary_guidelines.md).

### About Extension Publish

There is a pipeline to automatically build, upload and publish extension wheels.  
Once your PR is merged into master branch, a new PR will be created to update `src/index.json` automatically.  
The precondition is to put your code inside this repo and upgrade the version in the PR but do not modify `src/index.json`.

### CODE REVIEW

**Comment by yonzhan** [comment]
AKS

**Comment by qwordy** [comment]
Dangerous default value [] as argument (dangerous-default-value)

File: src/aks-preview/azext_aks_preview/custom.py [hunk 1]
```diff
@@ -2098,18 +2098,9 @@ def aks_agentpool_add(cmd,      # pylint: disable=unused-argument,too-many-local
                            "use 'aks nodepool list' to get current list of node pool".format(nodepool_name))
 
     upgradeSettings = AgentPoolUpgradeSettings()
-    taints_array = []
+    taints_array = [taint.strip() for taint in node_taints.split(',')]
```
Comment (qwordy) [inline src/aks-preview/azext_aks_preview/custom.py:2101]: `node_taints` is a list. Does a list has `split` method?
```
AttributeError: 'list' object has no attribute 'split'
```

Comment (paulgmiller) [reply src/aks-preview/azext_aks_preview/custom.py:2101]: Whoops yeah default value should be ""

**Comment by yungezz** [comment]
hi @paulgmiller @arrownj what's status of the PR? is it still valid?

**Comment by yungezz** [comment]
hi @paulgmiller  are you still on this PR? will you update it? Or is it ok to close it for now?

**Comment by yungezz** [comment]
hi @paulgmiller I'll close the PR for now, feel free to create new PR when ready. thanks :)

**Pull request closed without merging**